code/modules/research/designs/medical_designs.dm

```diff
@@ -395,6 +395,28 @@
 	category = list("Misc", "Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
 
+/datum/design/cyberimp_joywire
+	name = "Joywire Implant"
+	desc = "A widely popular (and addictive) implant that stimulates the brain's pleasure centers. Dramatically increases mood, but interferes with taste reception even if uninstalled."
+	id = "ci-joywire"
+	build_type = PROTOLATHE | MECHFAB
+	construction_time = 60
+	materials = list(/datum/material/iron = 600, /datum/material/glass = 600, /datum/material/gold = 500, /datum/material/silver = 500)
+	build_path = /obj/item/organ/cyberimp/brain/joywire
+	category = list("Misc", "Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+
+/datum/design/cyberimp_mindscrew
```

Comment:
addendum: I think this should not be readily available, notably because it's illegal and aftermarket

Reply:
just thought about it. Have it as a organ installed in corpses of most humanoid hostile mobs????

Reply:
also there is a blackmarket pr open this would fit well in